Dockerfile

```diff
@@ -1,4 +1,52 @@
+FROM golang:1.15.1 AS builder
+
+# no need to include cgo bindings
+ENV CGO_ENABLED=0 GOOS=linux GOARCH=amd64
+
+# add ca certificates and timezone data files
+# hadolint ignore=DL3008
+RUN apt-get install --yes --no-install-recommends ca-certificates tzdata
+
+# add unprivileged user
+RUN adduser --shell /bin/true --uid 1000 --disabled-login --no-create-home --gecos '' app \
+  && sed -i -r "/^(app|root)/!d" /etc/group /etc/passwd \
+  && sed -i -r 's#^(.*):[^:]*$#\1:/sbin/nologin#' /etc/passwd
+
+# this is where we build our app
+WORKDIR /go/src/app/
+
+# download and cache our dependencies
+VOLUME /go/pkg/mod
+COPY go.mod go.sum ./
+RUN go mod download
+
+# compile kubeaudit
+COPY . ./
+RUN go build -a -ldflags '-w -s -extldflags "-static"' -o /go/bin/kubeaudit ./cmd/ \
```

Comment:
goreleaser will already provide the pre-compiled binary here, you don't need to compile it again (unless you really want to).

e.g. https://github.com/goreleaser/goreleaser/blob/master/Dockerfile#L16

Reply:
I guess it's a double edged knife; on one side you want to simplify things and integrate with goreleaser, on the other side you also want to provide a full-docker build chain for people not needing/wanting Go on their system.

Reply:
In some projects I create a `goreleaser.dockerfile` just for goreleaser... but yeah, I get it, totally your call :)